stacks/services.yml

```diff
@@ -124,7 +124,7 @@ services:
   # CLI - Used for all console commands and tools.
   cli:
     hostname: cli
-    image: ${CLI_IMAGE:-docksal/cli:php7.4-3.0}
+    image: ${CLI_IMAGE:-docksal/cli:php8.1-3.2}
```

Comment:
do we think this might introduce a breaking change where people do not set the CLI_IMAGE per project and are running `fin start`? Considering in most of our stacks we are still defaulting to 7.4

Reply:
Added a note about switching to PHP 8.1 as the default and a link to docs explaining how to switch/pin to a particular version.